app/views/projects.html

```diff
@@ -25,9 +25,13 @@ <h2 class="text-center">Loading...</h2>
                         <h1>My Projects</h1>
                         <div class="projects-options">
                           <div class="projects-add" ng-if="canCreate">
-                            <a href="create-project" class="btn btn-md btn-primary">
+                            <button ng-click="createProject()" class="btn btn-md btn-primary">
+                              <span class="fa fa-plus"></span>
                               Create Project
-                            </a>
+                            </button>
+                            <origin-modal-popup shown="showNewProjectPanel" modal-title="Create Project" on-close="closeNewProjectPanel()">
```

Comment:
I could change either of the components but that would need to be done there and not here. Most of the components bind functions using < or =, I used & for the new origin-modal-popup component. Should I change that?